### PR TITLE
Display a simple game list panel

### PR DESCRIPTION
**Pull request by atilbian**
- [x] Create a GameList component
- [x] Create a basic data provider with a getList function to fetch from our api
- [x] Store our api url into a env var

## Demo
demo-simple-games-panel

### CODE REVIEW

File: .github/workflows/deploy-admin-app-s3.yml [hunk 1]
```diff
@@ -22,6 +22,9 @@ jobs:
       - name: Build
         working-directory: ./admin-app
         run: yarn build
+      - name: Create env file
+        working-directory: ./admin-app
+        run: echo "$VITE_HEX_ADMIN_API_URL" > .env
```
Comment (slax57) [inline .github/workflows/deploy-admin-app-s3.yml]: You're mixing up the definition of **one** env var with the definition of a whole `.env` file.

I suggest stg like `echo "$ADMIN_ENV_VARS" > .env`
with `ADMIN_ENV_VARS` containing something like:

```
VITE_HEX_ADMIN_API_URL=https://d1iqp7uyjrmy2a.cloudfront.net/api/admin
```

File: admin-app/src/components/GameList.tsx [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+interface GameListProps {}
+
+export const GameList = (props: GameListProps) => (
+  <List {...props}>
+    <Datagrid rowClick="edit">
+      <TextField source="id" />
+      <DateField source="createdAt" />
+      <DateField source="updatedAt" />
+      <NumberField source="player1.id" />
+      <NumberField source="player2.id" />
```
Comment (slax57) [inline admin-app/src/components/GameList.tsx:13]: Can you confirm you did plan to add the 'status' column later?

File: admin-app/src/data-provider.ts [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+    const url = `${apiUrl}/${resource}/`;
+
+    return httpClient(url).then(({ json }) => ({
+      data: json,
+      total: json.length,
+    }));
+  },
+
+  getOne: (resource: any) => {
+    throw Error();
```
Comment (slax57) [inline admin-app/src/data-provider.ts]:
```suggestion
    throw Error('This function is not yet implemented');
```

And I suggest making this replacement for all unimplemented methods of this file of course 😉

File: .github/workflows/deploy-admin-app-s3.yml (hunk 1 above)
Comment (atilbian) [reply .github/workflows/deploy-admin-app-s3.yml]: Oupsy doupsy !

File: admin-app/src/components/GameList.tsx (hunk 2 above)
Comment (atilbian) [reply admin-app/src/components/GameList.tsx:13]: Yep, but i can add it now too !

**Review by slax57 — APPROVED** [review]

File: admin-app/src/components/GameList.tsx [hunk 4]
```diff
@@ -0,0 +1,17 @@
+import * as React from 'react';
+import { List, Datagrid, TextField, DateField, NumberField } from 'react-admin';
+
+interface GameListProps {}
+
+export const GameList = (props: GameListProps) => (
+  <List {...props}>
```
Comment (djhi) [inline admin-app/src/components/GameList.tsx:7]: You don't need to pass props anymore in v4

File: admin-app/src/data-provider.ts [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+const apiUrl = import.meta.env.VITE_HEX_ADMIN_API_URL;
+const httpClient = fetchUtils.fetchJson;
+
+export default {
+  getList: (resource: string) => {
+    const url = `${apiUrl}/${resource}/`;
+
+    return httpClient(url).then(({ json }) => ({
+      data: json,
+      total: json.length,
```
Comment (djhi) [inline admin-app/src/data-provider.ts:12]: This is probably wrong but I suppose it's temporary

**Pull request merged**